takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -5,68 +5,65 @@
-          <img src="{% static 'img/logos/ap-white.png' %}" class="img-responsive" alt="africapractice">
-        </a>
-        <a href="https://codeforafrica.org" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/cfafrica-white.png' %}" class="img-responsive" alt="Code for Africa">
-        </a>
-        <a href="https://ihub.co.ke/" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/ihub-white.png' %}" class="img-responsive" alt="iHub">
-        </a>
+    <div class="row pt-5">
+      <div class="col-12">
+        <p class="mb-0 text-truncate font-weight-bold">An initiative by </p>
+        <div class="row pl-1">
+          <a href="http://www.africapractice.com/" target="_blank" rel="noopener"
+              class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+            <img src="{% static 'img/logos/ap-footer-white.png' %}" class="img-responsive" alt="africapractice">
+          </a>
+          <a href="https://codeforafrica.org" target="_blank" rel="noopener"
+              class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+            <img src="{% static 'img/logos/cfafrica-white.png' %}" class="img-responsive" alt="Code for Africa">
+          </a>
+          <a href="https://ihub.co.ke/" target="_blank" rel="noopener"
+              class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+            <img src="{% static 'img/logos/ihub-white.png' %}" class="img-responsive" alt="iHub">
+          </a>
+          <div class="col-4">
+          <p class="mb-0 pt-0 text-truncate text-center" style="opacity: 0.6; font-size: 16px;margin-top:-1.5rem;">Sponsored by:</p>
+            <a href="https://www.gatesfoundation.org" target="_blank" rel="noopener"
+              class="col ml-4 partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+              <img src="{% static 'img/logos/bmgf-white.png' %}" class="img-responsive" alt="Bill and Melinda Gates Foundation">
+            </a>
+          </div>
+        </div>
       </div>
     </div>
-    <div class="col-4">
-      <p class="mb-0 text-truncate text-center" style="opacity: 0.6; font-size: 16px">Sponsored by:</p>
-      <a href="https://www.gatesfoundation.org" target="_blank" rel="noopener"
-          class="col-8  ml-4 partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-        <img src="{% static 'img/logos/bmgf-white.png' %}" class="img-responsive" alt="Bill and Melinda Gates Foundation">
-      </a>
-    </div>
-  </div>
 </div>
-<style>
-ul li u {
-  color:#7ab1f2;
-}
-</style>
```

Comment:
Where is this defined to ensure it remains?

Reply:
Um..just moved it to the dropdown.html file so that it applys to all list items on the navbar